jr-extension-datetime/README.md

```diff
@@ -0,0 +1,28 @@
+## Overview
+
+This module extends the functionalities of jackson-jr adding support for LocalDateTime.
```

Comment:
And as per above, intent is to eventually support more Java 8 date/time types I think; not just this one.

Reply:
Okay, I have made the changes in readme. Now everything looks good 😄